synapse/app/homeserver.py

```diff
@@ -140,37 +136,15 @@ def _listener_http(
         else:
             root_resource = OptionsResource()
 
-        site = SynapseSite(
-            "synapse.access.%s.%s" % ("https" if tls else "http", site_tag),
-            site_tag,
```

Comment:
`site_tag` seems unused except for here, so I think we can remove declaring it above.

Reply:
It is used here still: https://github.com/matrix-org/synapse/blob/ac5184f7dd44ce1c2f5f4b2059e43f1f4651f84b/synapse/app/homeserver.py#L105-L108